test/svm/SvmSpoke.Fill.ts

```diff
@@ -631,4 +649,107 @@ describe("svm_spoke.fill", () => {
     assertSE(event.messageHash, new Uint8Array(32), `MessageHash should be zeroed`);
     assertSE(event.relayExecutionInfo.updatedMessageHash, new Uint8Array(32), `UpdatedMessageHash should be zeroed`);
   });
+
+  describe("codama client and solana kit", () => {
+    it("Fills a V3 relay and verifies balances with codama client and solana kit", async () => {
+      const rpcClient = createDefaultSolanaClient();
+      const signer = await createSignerFromKeyPair(await createKeyPairFromBytes(relayer.secretKey));
+
+      const [eventAuthority] = await getProgramDerivedAddress({
+        programAddress: address(program.programId.toString()),
+        seeds: ["__event_authority"],
+      });
+
+      let recipientAccount = await getAccount(connection, recipientTA);
+      assertSE(recipientAccount.amount, "0", "Recipient's balance should be 0 before the fill");
+
+      let relayerAccount = await getAccount(connection, relayerTA);
+      assertSE(relayerAccount.amount, seedBalance, "Relayer's balance should be equal to seed balance before the fill");
+
+      const relayHash = Array.from(calculateRelayHashUint8Array(relayData, chainId));
+
+      const formattedAccounts = {
+        state: address(accounts.state.toString()),
+        instructionParams: address(program.programId.toString()),
+        mint: address(mint.toString()),
+        relayerTokenAccount: address(relayerTA.toString()),
+        recipientTokenAccount: address(recipientTA.toString()),
+        fillStatus: address(accounts.fillStatus.toString()),
+        tokenProgram: address(TOKEN_PROGRAM_ID.toString()),
+        associatedTokenProgram: address(ASSOCIATED_TOKEN_PROGRAM_ID.toString()),
+        systemProgram: address(anchor.web3.SystemProgram.programId.toString()),
+        program: address(program.programId.toString()),
+        eventAuthority,
+        signer,
+        remainingAccounts: fillRemainingAccounts,
```

Comment:
this is not in `FillRelayAsyncInput` props

Reply:
You are right, remaining accounts are passed here:
https://github.com/across-protocol/contracts/blob/583d129037f51f5f3558fa00966b88a99399b2b7/test/svm/SvmSpoke.Fill.ts#L726-L730

And I forgot to remove it from this obj.